src/algebraic_geometry/is_open_comap_C.lean

```diff
@@ -7,6 +7,10 @@ import algebraic_geometry.prime_spectrum
 import ring_theory.polynomial.basic
 /-!
 The morphism `Spec R[x] --> Spec R` induced by the natural inclusion `R --> R[x]` is an open map.
+
+The main result if the part of the statement of Lemma 00FB in the Stacks Project.
```

Comment:
There is a typo in this sentence.

Reply:
Also, at least until we come up with an alternative definitive scheme, could I suggest use the full URL when linking to Stacks?

It's easy to grep for if we want to collect them all, and immediately clickable links are always nice.

Reply:
Johan, I corrected the error.

Scott, I already had placed a full URL: did you mean to add a link to the main page of the Stacks project?

Reply:
I cannot believe that I had missed another error in the sentence!

Thanks Johan for pointing it out!